xen/arch/arm/coproc/coproc.h

```diff
@@ -89,7 +185,6 @@ struct coproc_ops {
 
 /* vcoproc read/write operation context */
 struct vcoproc_rw_context {
-    struct coproc_device *coproc;
     struct hsr_dabt dabt;
     uint32_t offset;
```

Comment:
I think no, it is just a diff between addr and base within the device mmio range